Updated SalesForce to replace dead link, add hardware support

jadchaar opened the pull request:
Updated SalesForce entry to replace the dead doc link with an updated version. Added hardware support.

jadchaar committed:
Updated SalesForce to replace dead link, add hardware support

Updated SalesForce entry to replace the dead doc link with an updated version. Added hardware support.

mxxcon commented on _data/communication.yml:
```diff
@@ -162,7 +162,8 @@ websites:
       email: Yes
-      doc: https://help.salesforce.com/HTViewHelpDoc?id=security_require_two_factor_authentication.htm
+      hardware: Yes
+      doc: https://help.salesforce.com/articleView?id=security_overview_2fa.htm&type=5
```
Is there a better link for end-users rather than admins? Perhaps https://trust.salesforce.com/en/security/2fa/ ?

jadchaar replied:
Good point. My only problem with the article you linked is that it seems to only promote their proprietary application, rather than an authenticator app in general. Thoughts on that?

jadchaar replied:
The only other relevant link I can find is a developer page, but that is even more cryptic. A bunch of the others just redirect to the admin page.

mxxcon replied:
I personally don't have too much of a problem with them promoting it. People who know the difference will be able to figure it out on their own. Otherwise, it should simplify support issues for Salesforce.

jadchaar replied:
Hi @mxxcon I think I found a good one: https://developer.salesforce.com/docs/atlas.en-us.identityImplGuide.meta/identityImplGuide/add_time-based_token.htm

mxxcon replied:
this link looks better 👍

jadchaar committed:
Updated Salesforce link

New link better communicates the idea of how to setup an authenticator rather than an admin page.

mxxcon commented on _data/communication.yml:
```diff
@@ -162,7 +162,8 @@ websites:
       software: Yes
       email: Yes
-      doc: https://help.salesforce.com/HTViewHelpDoc?id=security_require_two_factor_authentication.htm
+      hardware: Yes
```
wait a sec, why did you add hardware? we consider mobile authenticator apps to still be "software".
"hardware" are physical fobs or yubikey and similar type devices

Carlgo11 replied:
They support U2F

jadchaar replied:
Yup I saw on that admin page that they supported U2F :)
